internal/status/status.go

```diff
@@ -60,6 +61,33 @@ func (s Status) GiveClassName() string {
 	return "Unassigned"
 }
 
+func CodeClassFromName(name string) (int, bool) {
+	switch strings.ToLower(name) {
+	case "1xx":
+		fallthrough
+	case "informal":
+		return Informal, true
+	case "2xx":
+		fallthrough
+	case "successfull":
+		return Successfull, true
```

Comment:
ℹ️ Change as follows once #7 or #4 are merged:

```suggestion
	case "successful":
		return Successful, true
```